src/System.Diagnostics.FileVersionInfo/src/System.Diagnostics.FileVersionInfo.builds

```diff
@@ -9,6 +9,11 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.FileVersionInfo.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netcore50</TargetGroup>
```

Comment:
I realize we're not at this point yet, but presumably our Windows-specific tests won't work with this new version.  Just something to keep in mind.